index.html

```diff
@@ -5,12 +5,12 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>OSCEBossKey</title>
-    <link href="https://fonts.googleapis.com/css?family=Nova+Round|Roboto:400,900" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Nova+Round|Roboto:400,700,900" rel="stylesheet">
```

Comment:
Do we know if loading four fonts will have a noticeable effect on initial loading time?